sql/database.sql

```diff
@@ -719,6 +721,7 @@ CREATE TABLE `codes` (
   `active` TINYINT(1) DEFAULT 1 COMMENT '0 = inactive, 1 = active',
   `reportable` TINYINT(1) DEFAULT 0 COMMENT '0 = non-reportable, 1 = reportable',
   `financial_reporting` TINYINT(1) DEFAULT 0 COMMENT '0 = negative, 1 = considered important code in financial reporting',
+  `revenue_code` varchar(255) NOT NULL default '',
   PRIMARY KEY  (`id`),
   KEY `code` (`code`),
   KEY `code_type` (`code_type`)
```

Comment:
Note that these entries do not have COMMENTS while the entries int he upgrade script do.